addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/BigRedirectsScannerTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2018 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.IOException;
+import net.htmlparser.jericho.Source;
+import org.apache.commons.httpclient.URI;
+import org.junit.Before;
+import org.junit.Test;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+
+public class BigRedirectsScannerTest extends PassiveScannerTest<BigRedirectsScanner> {
+    private static final String URI = "http://example.com";
+
+    protected HttpMessage msg;
+
+    @Override
+    protected BigRedirectsScanner createScanner() {
+        return new BigRedirectsScanner();
+    }
+
+    @Test
+    public void GivenRedirectWithSmallBodyThenItRaisesNoAlert() {
```

Comment:
Test method names should use standard camelCase.
Tests should include `//Given`, `//When`, and `//Then` comments.

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#unit-test-specific-guidelines